airbyte-server/src/main/java/io/airbyte/server/ServerApp.java

```diff
@@ -201,8 +201,8 @@ public static ServerRunnable getServer(final ServerFactory apiFactory,
     final TrackingClient trackingClient = TrackingClientSingleton.get();
     final JobTracker jobTracker = new JobTracker(configRepository, jobPersistence, trackingClient);
 
-    final WorkflowServiceStubs temporalService = TemporalUtils.createTemporalService(configs.getTemporalHost());
-    final TemporalClient temporalClient = TemporalClient.production(configs.getTemporalHost(), configs.getWorkspaceRoot(), configs);
+    final WorkflowServiceStubs temporalService = TemporalUtils.createTemporalProductionService();
```

Comment:
`TemporalUtils` already has a static `EnvConfigs` that it uses to access configs. Instead of passing the `temporalHost` in from the outside, `createTemporalProductionService` has internal logic to use either the `temporalHost` or the `temporalCloudHost` from `configs` based on whether or not `configs.temporalCloudEnabled` is true or not